augur/tasks/util/redis_list.py

```diff
@@ -168,8 +168,10 @@ def pop(self, index: int = None):
         """
 
         if index is None:
-
-            redis.rpop(self.redis_list_key)
+            # This will get a random index from the list and remove it, 
+            # decreasing the likelihood of everyone using the same key all the time
+            #redis.rpop(self.redis_list_key)
+            redis.spop(self.redis_list_key)
 
```

Comment:
This won't do anything because this isn't how we are getting the api keys from redis.